GeofenceDisplay/test/geofence-display.test.js

```diff
@@ -0,0 +1,34 @@
+let Leaflet = require('./verify/leaflet')
+require('./verify/L.WatermarkControl')
+
+describe('leaflet', () => {
+  test('happy', () => {
+    expect(Leaflet).toBeDefined()
+  })
+
+  test('leaflet-map', () => {
+    let map = L.map(document.createElement('div'))
+    expect(map).toBeDefined()
+  })
+
+  test('plugin-happy', () => {
```

Comment:
replace plugin with the name of the plugin, as per above

Reply:
👍